README.md

```diff
@@ -43,7 +43,7 @@
 
 ### Key Features <a name="key-features"></a>
 
-- **[The project was built using webpack]**
+- **[The project was built using webpack bundling]**
 - **[It was also built using ES6]**
 - **[Modules were imported/exported for the JavaScript]**
 
```

Comment:
- [ ] [optional] @Uthmanbello great work on your app, it would be nice if the future feature section was properly styled as you did in other sections.